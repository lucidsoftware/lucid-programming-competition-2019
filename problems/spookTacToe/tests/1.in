9 9
.........
.o.......
.#.......
#..o.....
x#.......
##......#
...#....x
o#o...#..
.o##..x#.