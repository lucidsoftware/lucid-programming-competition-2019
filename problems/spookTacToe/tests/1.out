oxoxoxoxo
xooxoxoxo
x#xoxoxox
#oxoxoxox
x#oxoxoxo
##oxoxox#
oxx#xoxox
o#ooxo#ox
xo##oxx#o